generator/Writer/Filters.php

```diff
@@ -11,7 +11,29 @@ public static function doc($text, array $options = []): string
 
         $lineLength = 80 - 3 - ($indentation * 4);
 
-        $text = str_replace('<br />', '', $text);
+        // Remove HTML breaks
+        $text = str_replace(['<br/>', '<br />'], '', $text);
+        $text = str_replace('&amp;', '&', $text);
```

Comment:
After checking the generated files there are some more chars to replace:
```
&lt; <
&gt; >
&quot; "
```

https://github.com/spatie/schema-org/pull/141/files#diff-3e5c3750f7d2fba6d1f88da5e0142ad9R1153